benchmarks/pom.xml

```diff
@@ -5,7 +5,7 @@
   <parent>
     <groupId>org.locationtech.geomesa</groupId>
     <artifactId>geomesa_2.11</artifactId>
-    <version>3.1.0-SNAPSHOT</version>
+    <version>3.2.0-SNAPSHOT</version>
```

Comment:
Hmm.... is behind a profile?

Reply:
yeah, we should update the release docs to include it

Reply:
i'll do that